src/lib/falcor/routes/staff/database-calls.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Fetches direct meta data of staff from the staff database table
+ * @param {any} database - The Knex object for the databse to update
+ * @param {string} queryField - Indicates which field to query by
+ * @param {string[]} queryParams - Array of parameters of type queryField of staff to fetch
+ * @param {string[]} columns - Which columns of the staff table to fetch
+ * @returns {Promise<Object[]>}
+ */
+export function staffQuery(database, queryField, queryParams, columns) {
```

Comment:
Make it async :)

Reply:
This is better than async, it returns a promise directly